### PR TITLE
Javascript javascript3 week3

### PR DESCRIPTION
**Pull request by 27ManishaPatel**

### CODE REVIEW

File: javascript/javascript3/week3/githubRepo/script.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      const repo = data.items.forEach((name) => {
+        const li = document.createElement('li')
+       li.innerHTML = `${name.name} : ${name.git_url}`
+        li.className = 'users-repo-url'
+        ul.appendChild(li);
+      })
+    })
+}
+async function toGetRepo() {
+  await Promise.all([
```
Comment (andrii1) [inline javascript/javascript3/week3/githubRepo/script.js:23]: nice with Promise.all

File: javascript/javascript3/week3/shoppingCart/script.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        this.products.forEach(pd => {
+            const ul = document.createElement("ul");
+            ul.innerHTML = ` Product Name: ${pd.name}`
+            shoppingCart.appendChild(ul);
+            const li = document.createElement("li");
+            li.innerHTML = `Price: ${pd.price}`
+            ul.appendChild(li)
+        })
+        const userDetail = document.createElement("p");
+        console.log(this.getUser(1))
```
Comment (andrii1) [inline javascript/javascript3/week3/shoppingCart/script.js:52]: delete console.log in final code :)

File: javascript/javascript3/week3/shoppingCart/script.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        userDetail.innerHTML = `UserName: ${this.getUser(2)}, Total Price: ${this.getTotal()}`;
+        shoppingCart.appendChild(userDetail);
+    }
+    getUser(user) {
+        fetch(`https://jsonplaceholder.typicode.com/users/${user}`)
+            .then(Response => Response.json())
+            .then(data => {
+
+                console.log(data.username)
```
Comment (andrii1) [inline javascript/javascript3/week3/shoppingCart/script.js:61]: also here delete console.log

File: javascript/javascript3/week3/shoppingCart/script.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+shoppingCart.addProduct(mobile);
+const iphone = new Product("I-phone", 8000);
+shoppingCart.addProduct(iphone);
+shoppingCart.removeProduct(mobile);
+shoppingCart.getTotal();
+shoppingCart.renderProducts();
+// Assuming dkr as default currency
+const plant = new Product("plant", 50);
+shoppingCart.addProduct(plant);
+console.log(plant.convertToCurrency("USD")); // 7.5
```
Comment (andrii1) [inline javascript/javascript3/week3/shoppingCart/script.js:80]: also here

File: javascript/javascript3/week3/shoppingCart/script.js (hunk 2 above)
Comment (27ManishaPatel) [reply javascript/javascript3/week3/shoppingCart/script.js:52]: Yes I always check in console and then forget to delete it 😃  but will do that

**Pull request merged**